src/components/vm/disks/diskEdit.jsx

```diff
@@ -126,7 +126,7 @@ const AccessRow = ({ onValueChanged, dialogValues, diskDevice, driverType, idPre
                        name="access"
                        value="writable"
                        isChecked={dialogValues.access == "writable" }
-                       onChange={event => {
+                       onChange={(event, _) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230704-092041-0b751cdc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)